### ISSUE TITLE
FFmpegInterop for games

### ISSUE THREAD

**Issue by 1d8enchanted** [Microsoft/FFmpegInterop#98]
Is it possible to use this for DirectX project (a game) to "directly" render video (without XAML involved)? Just to take a buffer, something like this:

unsigned char* pixels = f->getBuffer();  // I would reeally like to get pixels to unsigned char*

and then to use pixels as I want

**Comment by khouzam** [comment]
Yes, if you look at the way FFmpeg is used, you can do a similar approach to the uncompressed way of decoding and get the frames that way. Another thing that you can try is to use the [MediaPlayer](https://msdn.microsoft.com/en-us/library/windows/apps/windows.media.playback.mediaplayer.aspx) class, this has a [GetSurface] (https://msdn.microsoft.com/en-us/library/windows/apps/windows.media.playback.mediaplayer.getsurface.aspx) method that you can use to get the bits and display them without relying on XAML, but it still can connect to a MediaStreamSource.

One of the advantages of this approach is that audio and video playback should still be synchronized.
